apps/docs/spec/supabase_py_v2.yml

```
@@ -4409,6 +4409,241 @@ functions:
+            .channel('room1')
+            .on_postgres_changes("UPDATE", schema="public", table="countries", callback=handle_record_updated)
+            .subscribe()
+          ```
+      - id: listen-to-deletes
+        name: Listen to deletes
+        description: |
+          By default, Supabase does not send deleted records. If you want to receive the deleted record you can
+          enable full replication for the table you are listening too:
+
+          ```sql
+          alter table "your_table" replica identity full;
+          ```
+        code: |
+          ```python
+          supabase
+            .channel('room1')
+            .on_postgres_changes("DELETE", schema="public", table="countries", callback=handle_record_deleted)
+            .subscribe()
+          ```
+      - id: listen-to-multiple-events
+        name: Listen to multiple events
+        description: You can chain listeners if you want to listen to multiple events for each table.
+        code: |
+          ```python
+          supabase
+            .channel('room1')
+            .on_postgres_changes("INSERT", schema="public", table="countries", callback=handle_record_inserted)
+            .on_postgres_changes("DELETE", schema="public", table="countries", callback=handle_record_deleted)
+            .subscribe()
+          ```
+      - id: listening-to-row-level-changes
+        name: Listen to row level changes
+        description: You can listen to individual rows using the format `{table}:{col}=eq.{val}` - where `{col}` is the column name, and `{val}` is the value which you want to match.
+        notes: |
+          - ``eq`` filter works with all database types as under the hood, it's casting both the filter value and the database value to the correct type and then comparing them.
+        code: |
+          ```python
+          supabase
+            .channel('room1')
+            .on_postgres_changes("UPDATE", schema="public", table="countries", filter="id=eq.200", callback=handle_record_updated)
+            .subscribe()
+          ```
+  - id: broadcast-message
+    title: broadcastMessage()
+    description: |
+      Broadcast a message to all connected clients to a channel.
+    notes: |
+      - When using REST you don't need to subscribe to the channel
+      - REST calls are only available from 2.37.0 onwards
+    $ref: '@supabase/realtime-js.RealtimeChannel.send'
+    examples:
+      - id: send-a-message
+        name: Send a message via websocket
+        isSpotlight: true
+        code: |
+          ```python
+          channel = supabase.channel('room1')
+
+          def on_subscribe(status, err):
+            if status == RealtimeSubscribeStates.SUBSCRIBED:
+              channel.send_broadcast('cursor-pos', { "x": random.random(), "y": random.random() })
+
+          channel.subscribe(on_subscribe)
+          ```
+        response: |
+          ```
+          ok | timed out | error
+          ```
+      # - id: send-a-message-via-rest
```

Don't forget to remove before merging
```suggestion
      # - id: send-a-message-via-rest
```